docs/sidebarsGenAI.ts

```diff
@@ -264,62 +333,6 @@ const sidebarsGenAI: SidebarsConfig = {
             id: 'tracing/integrations/index',
           },
         },
-        {
-          type: 'category',
-          label: 'Guides',
-          items: [
-            {
-              type: 'doc',
-              id: 'tracing/track-users-sessions/index',
-              label: 'Track Users and Sessions',
-            },
-            {
-              type: 'doc',
-              id: 'tracing/collect-user-feedback/index',
-              label: 'User Feedback Collection',
-            },
-            {
-              type: 'doc',
-              id: 'tracing/attach-tags/index',
-              label: 'Tagging Traces',
-            },
-            {
-              type: 'doc',
-              id: 'tracing/observe-with-traces/delete-traces',
-            },
-            {
-              type: 'doc',
-              id: 'tracing/track-environments-context/index',
-              label: 'Track Application Versions',
-            },
-            {
-              type: 'doc',
-              id: 'tracing/observe-with-traces/masking',
-              label: 'Redacting Sensitive Data',
-            },
-            {
-              type: 'doc',
-              id: 'tracing/lightweight-sdk',
-              label: 'Lightweight Tracing SDK',
-            },
-          ],
-        },
-
-        {
-          type: 'category',
-          label: 'Viewing & Searching Traces',
-          items: [
-            {
-              type: 'doc',
-              id: 'tracing/observe-with-traces/ui',
-            },
-            {
-              type: 'doc',
-              id: 'tracing/search-traces',
-              label: 'Searching for Traces',
-            },
-          ],
-        },
         {
           type: 'category',
           label: 'OpenTelemetry',
```

Comment:
Happy to discuss but personally I feel it's unexpected that OpenTelemetry is at the same level as guides/integrations etc. We can probably combine it with https://pr-19329--mlflow-docs-preview.netlify.app/docs/latest/genai/tracing/app-instrumentation/opentelemetry/?